.github/workflows/pr_build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       name: '${{ github.event.repository.name }}'
       tags: pr-${{ github.event.number }},latest-rc
     secrets: inherit
-  trivy-scans:
-    if: (github.base_ref == 'develop' || github.base_ref == 'main' || github.base_ref == 'master' ) && github.event.pull_request.merged == false
-    uses: kbase/.github/.github/workflows/reusable_trivy-scans.yml@main
-    secrets: inherit
+#  trivy-scans:
+#    if: (github.base_ref == 'develop' || github.base_ref == 'main' || github.base_ref == 'master' ) && github.event.pull_request.merged == false
+#    uses: kbase/.github/.github/workflows/reusable_trivy-scans.yml@main
+#    secrets: inherit
```

Comment:
Why is this disabled?

Reply:
Because this repo fails these scans I believe

Reply:
Its been a while. We could try to re-enable it and see if it was a temporary thing or something broken in this repo

Reply:
it's just testing for the log4j bug right? That's easy to fix if so, I'd say reenable

Reply:
https://github.com/kbase/catalog/pull/152